<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewTab"/>
    <addaction name="actionNewWindow"/>
    <addaction name="separator"/>
    <addaction name="actionOpenFileManager"/>
    <addaction name="actionCloseTab"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNewTab">
   <property name="icon">
    <iconset theme="tab-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New &amp;Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="icon">
    <iconset theme="window-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New &amp;Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpenFileManager">
   <property name="icon">
    <iconset theme="system-file-manager">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open File Manager</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Close Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset theme="edit-find">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
